### DISCUSSION TITLE
MenuSection as Resource or MenuItem with Icon

### DISCUSSION THREAD

**Discussion by jjjrmy** [laravel/nova-issues#4020] [Ideas & Feature Requests]
- Nova Version: 4.2.1

### Description:
Right now you can do a MenuSection as a Dashboard with an Icon, but if you want a top-level resource it's not possible.

```php
MenuSection::dashboard(Main::class)->icon('chart-bar'),
```

You can add a top-level MenuItem for a Resource but can't add an Icon to it.

**Comment by davidhemphill** [comment]
In the next release there will be a `MenuSection::resource` method:

```php
MenuSection::resource(License::class)
    ->icon('exclamation-circle'),
```

For right now you can do this:

```php
MenuSection::make('Licenses')
    ->path('/resources/licenses')
    ->icon('document-text'),
```

**Reply by jjjrmy** [reply]
ETA?
